samples/apps/copilot-chat-app/README.md

```diff
@@ -0,0 +1,77 @@
+# Copilot Chat Sample Application
+>! IMPORTANT  This learning sample is for educational purposes only and should not be used in any production use case.  It is intended to highlight concepts of Semantic Kernel and not any architectural / Security design practices to be used.
+
+## About the Copilot 
+The Copilot Chat sample allows you to build your own integrated large language model chatbot.  This is an enriched intelligence app, with multiple dynamic components including command messages, user intent, and memories.  The chat prompt and response will evolve as the conversation between the user and the application proceeds.  This chat experience is a chat skill containing multiple functions that work together to construct the final prompt for each exchange.
+
+
+![UI Sample](images/UI-Sample.png)
+
+## Dependencies:
+
+Before following these instructions, please ensure your development environment and these components are functional:
+1.	For this sample application, it is assumed that you already have VS Code installed as your IDE.
+2.	As the repository is stored on GitHub, Git should be installed as well.  
+3.	The sample application requires .NET 6.0 or greater.  
+4.	Node.js should be installed, 
+5.	Yarn will be required.
+
+
+## Running the Sample
+1.	You will need an Open AI Key or Azure Open AI Service Key for this sample.
+2.	You will need to register your application in the Azure Portal.  Follow the steps to register your app here.
```

Comment:
"your application" is confusing in the context of our application that they are trying to use. Let's use the term "an application registration".

Reply:
Updated